qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/commons/snippets/SnippetRegistry.java

```diff
@@ -315,9 +315,42 @@ private static MarkupContent createDocumentation(Snippet snippet, Map<String, St
 			doc.append("```");
 			doc.append(System.lineSeparator());
 		}
+		addLinks(snippet.getLinks(), doc, canSupportMarkdown);		
 		return new MarkupContent(canSupportMarkdown ? MarkupKind.MARKDOWN : MarkupKind.PLAINTEXT, doc.toString());
 	}
 
+	public static void addLinks(List<Link> links, StringBuilder documentation, boolean canSupportMarkdown) {
+		if (links != null && !links.isEmpty()) {
+			documentation.append(System.lineSeparator());
+			documentation.append(System.lineSeparator());
+			documentation.append("See ");
+			for (int i = 0; i < links.size(); i++) {
+				Link link = links.get(i);
+				if (i > 0) {
+					if (i == links.size() - 1) {
+						documentation.append(" and ");
+					} else {
+						documentation.append(", ");
+					}
+				}
+				addLink(link.getUrl(), link.getLabel(), documentation, canSupportMarkdown);
+			}
+			documentation.append(" for more informations.");

```

Comment:
Same here: `" for more information."`

Reply:
fixed

Reply:
I changed the render  of links when there are several urls, I display then with bullet:

![image](https://user-images.githubusercontent.com/1932211/212881565-f89b4f3f-26a5-4ef9-9f50-0659d645c421.png)